app/views/admin/versions/_object_desc_and_link.html.haml

```diff
@@ -235,5 +235,5 @@
     - else
       (Organization Deleted)
   - else
-    in conference
-    = link_to_conference(version.conference_id)
+    //in conference
```

Comment:
How is it useful to have these commented out lines on our code base?

Reply:
@differentreality  I commented out the lines because I was not sure that will it pass all the test cases or not. If it would have passes, i would certainly remove the commented out lines.

Reply:
@Harry30897 Hey. You can locally check for the tests whether they are passing or not by running the command  ```bundle exec rspec```. And for checking the styling you can run the command ``` bundle exec rubocop -Dc .rubocop.yml```.

Reply:
Thankyou @namangupta01  Will take care of this from now onwards.